src/pages/page.js

```diff
@@ -76,14 +81,15 @@ function setGlobalCSSVars(props) {
 }
 // This component creates a snack bar alert if errorMsg is not ""
 export function ErrorHandler(props) {
-  var errorType = props.error[0];
-  var errorMsg = props.error[1];
+  var errorType = props.error[0]
```

Comment:
is this going to get reformatted again?

Reply:
I'm honestly not sure why it got reformatted, I can fix it